src/plugins/cams/datasource.py

```
@@ -0,0 +1,509 @@
+"""CAMS data source implementation for PM2.5 and atmospheric composition data."""
+
+import asyncio
+import logging
+import os
+from typing import List, Optional, Dict, Any, AsyncIterator
```

Ruff (0.12.2)

Import from `collections.abc` instead: `AsyncIterator`

Import from `collections.abc`

(UP035)